src/Facebook/InstantArticles/Transformer/Getters/DateGetter.php

```diff
@@ -10,50 +10,50 @@
 
 use Facebook\InstantArticles\Validators\Type;
 
-class DateGetter extends StringGetter
+class DateGetter extends AbstractGetter
 {
     /**
      * @var string
      */
-    protected $format;
+    protected ?string $format;
 
-    public function createFrom($properties)
+    public function createFrom(array<string, string> $properties): this
```

Comment:
Should we use `Map` instead of `array`? Either would work, but we're already using Hack collections in other places. No strong feelings, it's your call.

Reply:
The first find/replace generated tons of bugs, so moved back to array.
It would be better to use Map, for sure.